src/encoder.js

```diff
@@ -71,8 +64,11 @@ class Encoder {
    * @return The converted string to single padded-right hex string.
    */
   static stringToHex(string, maxCharLen) {
-    if (string === undefined) {
-      throw new Error(`string should not be undefined`);
+    if (!Validator.isString(string)) {
+      throw new Error(`string should be a String`);
```

Comment:
lodash has utility like this, for example _.isString().

Reply:
hmm the real question is... what does lodash not have lol. fixed.

Reply:
yeah they are there to fix javascript ...